File: 백준/Bronze/1546. 평균/평균.cc
#include <stdio.h>

int main()
{
    int N = 0;
    int score1[1000] = {0,};
    int max = 0;
    double avg = 0;
    double score2[1000] = {0.0,};
    scanf("%d", &N);
    
    for(int i = 0; i < N; i++)
    {
        int num = 0;
        scanf("%d", &num);
        score1[i] = num;
    }
    for(int i = 0; i < N; i++)
    {
        if(score1[i] > max)
        {
            max = score1[i];
        }
    }
    for(int i = 0; i < N; i++)
    {
         score2[i] = (double)score1[i]/max*100;  
         avg = score2[i] + avg;
    }
    printf("%.2lf", avg/N);
    return 0;
}

File: 백준/Bronze/1546. 평균/평균.java
import java.util.Scanner;

public class Main
{
    public static void main(String[] args)
    {
        Scanner scanner = new Scanner(System.in);
        int N = scanner.nextInt();
        int max = 0;
        int[] score1 = new int[N];
        double[] score2 = new double[N];
        double avg = 0;

        for(int i = 0; i < N; i++)
        {
            score1[i] = scanner.nextInt();
            if(score1[i] > max)
            {
                max = score1[i];
            }
        }

        for(int i = 0; i < N; i++)
        {
            score2[i] = ((double)score1[i] / max) * 100;
            avg += score2[i];
        }

        avg = avg / N;
        System.out.printf("%.2f", avg); 
        scanner.close();
    }
}